FarihaIS/VanRoomies
# Frontend Initialization + Login Screen

# Description

PR to set up an Android Studio project in the frontend folder. This PR also includes a basic Login Screen (not yet styled) with Google Sign-in integrated.

References Issue #9 & #12 

# How Has This Been Tested?
Manual testing via Android Studio / emulator
See CPEN321 tutorials for more information about emulator setup
- The user should be redirected to a page with a single textview with a placeholder message after logging in
- On subsequent launches of the app, the user should be sent directly to the textview activity
- A signout button will be added in a later PR.

All code for this PR were done in paired programming with @FarihaIS
